libs/blocks/text/text.js

```diff
@@ -117,4 +131,13 @@ export default function init(el) {
     lastActionArea.insertAdjacentElement('afterend', div);
     div.append(lastActionArea);
   }
+
+  const mnemonicList = el.querySelector('.mnemonic-list');
+  if (mnemonicList) {
+    const foreground = mnemonicList?.closest('.foreground');
+    if (foreground) {
+      mnemonicList.querySelectorAll('p').forEach((product) => product.removeAttribute('class'));
+      await loadMnemonicList(foreground);
+    }
+  }
```

Comment:
Nit: you might be able to avoid some nesting here to make it a bit easier to follow.

```suggestion
  const mnemonicList = el.querySelector('.mnemonic-list');
  const foreground = mnemonicList?.closest('.foreground');
  if (!foreground) return;
  mnemonicList.querySelectorAll('p').forEach((product) => product.removeAttribute('class'));
  await loadMnemonicList(foreground);
```